cpp/src/plasma/client.cc

```diff
@@ -384,6 +407,22 @@ Status PlasmaClient::Get(const ObjectID* object_ids, int64_t num_objects,
   return Status::OK();
 }
 
+Status PlasmaClient::Get(const ObjectID* object_ids, int64_t num_objects,
+                         int64_t timeout_ms, ObjectBuffer* object_buffers) {
+  const auto wrap_buffer = [](const ObjectID& object_id,
```

Comment:
Can we get rid of this callback, and do the wrapping into PlasmaBuffer as a post-processing step to GetBuffers?

Reply:
That sounds a bit brittle. You want to have a single `PlasmaBuffer` per exported (e.g. mmap'ed) piece of memory, but both data and metadata share that piece of memory.